src/main/java/uk/gov/ch/transformers/officer/bankrupt/BankruptOfficersTransformer.java

```diff
@@ -75,6 +75,7 @@ public ScottishBankruptOfficerSearchResult convertToSearchResult(ScottishBankrup
         details.setAddressLine1(scottishBankruptOfficerDetailsDataModel.getAddressLine1());
         details.setAddressLine2(scottishBankruptOfficerDetailsDataModel.getAddressLine2());
         details.setAddressLine3(scottishBankruptOfficerDetailsDataModel.getAddressLine3());
+        details.setDebtorDischargeDate(scottishBankruptOfficerDetailsDataModel.getDebtorDischargeDate());
```

Comment:
A small one but maybe put this at the end as it's in the middle of the address fields? :)